app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -60,7 +60,16 @@ const DiffViewer = React.createClass({
   },
 
   wikiUrl() {
-    return `https://${this.props.revision.wiki.language}.${this.props.revision.wiki.project}.org`;
+    const wikiProject = this.props.revision.wiki.project;
+    let url;
+
+    if (wikiProject === 'wikidata') {
+      url = `https://${this.props.revision.wiki.project}.org`;
+    } else {
+      url = `https://${this.props.revision.wiki.language}.${this.props.revision.wiki.project}.org`;
+    }
+
+    return url;
```

Comment:
This is more complicated than it needs to be. Maybe instead:

```javascript
if (this.props.revision.wiki.project === 'wikidata') { return 'https://www.wikidata.org' }
return `https://${this.props.revision.wiki.language}.${this.props.revision.wiki.project}.org`;
```

Reply:
Instead of that, you could make it a little more general by switching on language:

```javascript
if (this.props.revision.wiki.language) {
  return  `https://${this.props.revision.wiki.language}.${this.props.revision.wiki.project}.org`;
} else {
  return  `https://${this.props.revision.wiki.project}.org`;
}

That should correctly handle both wikidata and multilingual wikisource, the two wikis that have a project but no language.